Traceback (most recent call last):
  File "C:\ProgramData\Miniconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\ProgramData\Miniconda3\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import os
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.decomposition import TruncatedSVD
from sklearn.model_selection import train_test_split
import seaborn as sns
import surprise
from surprise import SVD, Dataset, Reader, accuracy
from surprise.model_selection import cross_validate
pd.set_option('display.max_columns', 500)
pd.set_option('display.width', 2000)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_3012/1338654878.py[0m in [0;36m<module>[1;34m[0m
[0;32m      6[0m [1;32mfrom[0m [0msklearn[0m[1;33m.[0m[0mmodel_selection[0m [1;32mimport[0m [0mtrain_test_split[0m[1;33m[0m[1;33m[0m[0m
[0;32m      7[0m [1;32mimport[0m [0mseaborn[0m [1;32mas[0m [0msns[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 8[1;33m [1;32mimport[0m [0msurprise[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      9[0m [1;32mfrom[0m [0msurprise[0m [1;32mimport[0m [0mSVD[0m[1;33m,[0m [0mDataset[0m[1;33m,[0m [0mReader[0m[1;33m,[0m [0maccuracy[0m[1;33m[0m[1;33m[0m[0m
[0;32m     10[0m [1;32mfrom[0m [0msurprise[0m[1;33m.[0m[0mmodel_selection[0m [1;32mimport[0m [0mcross_validate[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'surprise'
ModuleNotFoundError: No module named 'surprise'

